vmdb/app/views/container_route/show.html.haml

```diff
@@ -1 +1,4 @@
-= render :partial => @showtype
+- if %w(container_routes container_projects).include?(@display)
```

Comment:
Where do these strings come from?  I'm really concerned about including a list of magic strings.